CHANGELOG

```diff
@@ -7,6 +7,8 @@ Qtile 0.19.0, released 2021-12-22:
         - Add ability to draw borders to the Bar. Can customise size and colour per edge.
         - Add `StatusNotifier` widget implementing the `StatusNotifierItem` specification.
           NB Widget does not provide context menus.
+        - `StatusNotifier` widget added to default config for Wayland backends only
+          (SysTray remains default for X11 backends)
         - Add `total` bandwidth format value to the Net widget.
```

Comment:
Could you add another entry saying that defined widgets that are not compatible with the running backend will invisibly be removed? And perhaps a line in the docs below the widgets saying the same thing, so that anybody reading the "Only available on the following backends:" note on a widget see it and search the page to see what this means/what will happen, they'll see that they can just leave it in with no issue.

Reply:
Also, the diff doesn't actually make the change that this changelog entry describes - the default config is unchanged. Are we going with the backend check to set the systray to `SysTray` or `StatusNotifier`?

Reply:
Oops. Forgot to change the changelog.

Reply:
> Also, the diff doesn't actually make the change that this changelog entry describes - the default config is unchanged. Are we going with the backend check to set the systray to `SysTray` or `StatusNotifier`?

No. We're not doing this because the config is loaded before the backend is started so we can't check the backend. However, I will include `StatusNotifier` as a commented line instead,